### PR TITLE
Rating

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review my code for my Component Library's Rating Component.

The page includes:

Header with Girlifycoder Logo, Heading, and Github link
The navigation bar on the left
Sidebar on the right
Details about the Component and snippet

https://user-images.githubusercontent.com/65448358/160251133-60d89023-3ea9-4652-8ebe-acd9b3941158.mp4

### CODE REVIEW

File: rating/styles2-rating.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+.rating-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.rating-snippet {
+  margin: 1rem 0rem;
+}
+.standard-rating {
+  color: yellow;
```
Comment (pnchinmay) [inline rating/styles2-rating.css:27]: These give different values in different browsers
Use color variable with hex or rgb color code

**Review by pnchinmay — APPROVED** [review]
Left comments
Everything else looks good :100:

File: rating/styles2-rating.css (hunk 1 above)
Comment (khushijohri2001) [reply rating/styles2-rating.css:27]: done

**Review by neetukumari4858 — COMMENTED** [review]
overall looks good and Code is readable .

**Pull request merged**